imap_processing/swapi/l1/swapi_l1.py

```diff
@@ -470,9 +470,20 @@ def process_swapi_science(
     scem_compression_flags = process_sweep_data(good_sweep_sci, "scem_rng_st")
     coin_compression_flags = process_sweep_data(good_sweep_sci, "coin_rng_st")
 
-    swp_pcem_counts = decompress_count(raw_pcem_count, pcem_compression_flags)
-    swp_scem_counts = decompress_count(raw_scem_count, scem_compression_flags)
-    swp_coin_counts = decompress_count(raw_coin_count, coin_compression_flags)
+    swp_pcem_counts = decompress_count(raw_pcem_count, pcem_compression_flags).astype(
```

Comment:
If all of these are supposed to be float32 do you want to make decompress_count return a float32 array by casting it within the function return?